nix/services/elasticsearch.nix

```
@@ -147,8 +147,11 @@ in
                 export ES_HOME ES_JAVA_OPTS ES_PATH_CONF
 
                 # Install plugins
-                rm -f "${config.dataDir}/plugins"
-                ln -sf ${esPlugins}/plugins "${config.dataDir}/plugins"
+                rm -rf "${config.dataDir}/plugins"
+                cp -rL ${esPlugins}/plugins "${config.dataDir}/plugins"
+                find "${config.dataDir}/plugins" -type f -exec chmod 0700 {} \;
+                find "${config.dataDir}/plugins" -type d -exec chmod 0700 {} \;
```

I think we don't need to remove permissions from the group and other. The original perms on the directories are 555.
```suggestion
                find "${config.dataDir}/plugins" -type d -exec chmod u+x {} \;
```